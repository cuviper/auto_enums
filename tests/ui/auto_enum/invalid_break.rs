fn closure() -> impl Fn() {
    || break
}

fn async_block() -> impl std::future::Future<Output = ()> {
    async { break }
}

fn main() {}
