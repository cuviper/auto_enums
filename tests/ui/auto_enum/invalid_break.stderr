error[E0267]: `break` inside of a closure
 --> $DIR/invalid_break.rs:2:8
  |
2 |     || break
  |     -- ^^^^^ cannot `break` inside of a closure
  |     |
  |     enclosing closure

error[E0267]: `break` inside of an `async` block
 --> $DIR/invalid_break.rs:6:13
  |
6 |     async { break }
  |           --^^^^^--
  |           | |
  |           | cannot `break` inside of an `async` block
  |           enclosing `async` block
